src/app/app.module.ts

```diff
@@ -161,9 +162,10 @@ const maskConfig: Partial<IConfig> = {
       TechnologyEffects,
       ProjectTypeEffects,
       EntryEffects,
+      UsersEffects,
```

Comment:
You need to keep these two imports? `UsersEffects`  `UserEffects`

Reply:
Yes, both effects do different things.
```
import { UserEffects as UsersEffects } from './modules/users/store/user.effects';
import { UserEffects } from './modules/users/users/store/user.effects';
```
We found that there is a conflict with the names, especially in the `UserEffects` of users.

Should we change the name?